### ISSUE TITLE
CEF : jquery dblclick event not working

### ISSUE THREAD

**Issue by ihaswn** [multitheftauto/mtasa-blue#2963]
### Describe the bug

hi, the double click event is not working, everything is ok in external browser but the event is not getting called inside mta cef.

the event : https://api.jquery.com/dblclick/

### Steps to reproduce

use dblclick event, example code:
```
$("#an_element").dblclick(function() {
        $(this).hide();
});
```

### Version

client: Multi Theft Auto v1.5.9-release-21630 - windows 10, 64bit
server: MTA:SA Server v1.5.9-release-21476 - windows 10, 64bit

### Security Policy

- [X] I have read and understood the Security Policy and this issue is not security related.

**Comment by Xenius97** [comment]
There is a little trick, that could solve your problem temporary:
Create an onClientDoubleClick event in lua, and then invoke the browser like this:
```lua
addEventHandler("onClientDoubleClick", root, function(button)
	if button == "left" then
		executeBrowserJavascript(browser, "dblclick();")
	end
end)
```

JS (using jquery's example):
```js
var divdbl = $("div").first();
function dblclick(){
	if (!!divdbl.filter(":hover").length)
		divdbl.toggleClass("dbl");
}
```

But it's a valid problem, same on MTA 1.6 too.

**Comment by ihaswn** [comment]
yeah i already managed to do this, but with onClientGuiDoubleClick, its just a temprory solution it would be better to have the double click event from js. thanks for ur repsonde

**Comment by lopezloo** [comment]
It looks like `CEGUI::Window::EventMouseButtonDown` is not being called for the next click so that's why Chromium doesn't interpret it as double click. //edit: Tested and even if it would fire correctly it still wouldn't be interpreted by Chromium as double click so it requires passing click count to it as implemented in #3040.

Double click log from `CGUIWebBrowser_Impl::Event_MouseButtonDown` and `CGUIWebBrowser_Impl::Event_MouseButtonUp`:
```
VM7:1 CGUIWebBrowser_Impl::Event_MouseButtonDown BROWSER_MOUSEBUTTON_LEFT
VM8:1 CGUIWebBrowser_Impl::Event_MouseButtonUp BROWSER_MOUSEBUTTON_LEFT
VM9:1 CGUIWebBrowser_Impl::Event_MouseButtonUp BROWSER_MOUSEBUTTON_LEFT
```

Slow double click log from `CGUIWebBrowser_Impl::Event_MouseButtonDown` and `CGUIWebBrowser_Impl::Event_MouseButtonUp`:
```
VM7:1 CGUIWebBrowser_Impl::Event_MouseButtonDown BROWSER_MOUSEBUTTON_LEFT
VM8:1 CGUIWebBrowser_Impl::Event_MouseButtonUp BROWSER_MOUSEBUTTON_LEFT
VM9:1 CGUIWebBrowser_Impl::Event_MouseButtonDown BROWSER_MOUSEBUTTON_LEFT
VM10:1 CGUIWebBrowser_Impl::Event_MouseButtonUp BROWSER_MOUSEBUTTON_LEFT
```

Double click as seen by MTA events:
```
onClientGUIMouseDown left @1618231
onClientClick left down@1618231
onClientGUIMouseUp left @1618271
onClientGUIClick left up @1618272
onClientClick left up@1618272
onClientClick left down@1618311
onClientDoubleClick left
onClientGUIMouseUp left @1618371
onClientGUIClick left up @1618372
onClientClick left up@1618372
```
Notice that `onClientGUIMouseDown` isn't being called for next click.

Slow double click as seen by MTA events:
```
onClientGUIMouseDown left @1666929
onClientClick left down@1666929
onClientGUIMouseUp left @1666999
onClientGUIClick left up @1666999
onClientClick left up@1667000
onClientGUIMouseDown left @1667389
onClientClick left down@1667389
onClientGUIMouseUp left @1667449
onClientGUIClick left up @1667449
onClientClick left up@1667450
```
